packages/miniflare/src/plugins/core/errors/index.ts

```diff
@@ -247,19 +273,95 @@ export function reviveError(
 	return error;
 }
 
+export async function getScriptSource(inspectorURL: URL, moduleName: string) {
+	// Look for workers available in the inspector URL
+	const httpURL = `http://${inspectorURL.hostname}:${inspectorURL.port}/json`;
+	const response = await fetch(httpURL);
+	const workers = response.ok ? await response.json() : null;
+	// Find the first available worker
+	const worker = Array.isArray(workers) ? workers[0] : null;
+
+	if (!worker || !worker.webSocketDebuggerUrl) {
+		return;
+	}
+
+	// Connect to the specific worker's WebSocket
+	const promise = new DeferredPromise<string | undefined>();
+	const ws = new NodeWebSocket(worker.webSocketDebuggerUrl);
```

Comment:
Does this connecting to a Workers websocket URL cause issues with multiple clients/already active debugger sessions?

Reply:
Dropped native module support in [ae2bbd6](https://github.com/cloudflare/workers-sdk/pull/9971/commits/ae2bbd610fb6cd372f8ca539d33d9a8f96be8975)

Reply:
I have also updated the screenshot in the description to reflect the latest implementation